bin/compile

```diff
@@ -66,19 +73,20 @@ if [ ! `which pdflatex` ]; then
 
     PROF=$BIN_DIR/../conf/texlive.profile
 
-    echo "TEXDIR $TEXLIVE_HOME" >> $PROF 
+    echo "TEXDIR $TEXLIVE_HOME" >> $PROF
     echo "TEXMFCONFIG $TEXLIVE_HOME/var/texmf-config" >> $PROF
-    echo "TEXMFHOME $TEXLIVE_HOME/var/texmf" >> $PROF 
-    echo "TEXMFLOCAL $TEXLIVE_HOME/texmf-local" >> $PROF 
-    echo "TEXMFSYSCONFIG $TEXLIVE_HOME/texmf-config" >> $PROF 
-    echo "TEXMFSYSVAR $TEXLIVE_HOME/texmf-var" >> $PROF 
-    echo "TEXMFVAR $TEXLIVE_HOME/var/texmf-var" >> $PROF 
+    echo "TEXMFHOME $TEXLIVE_HOME/var/texmf" >> $PROF
+    echo "TEXMFLOCAL $TEXLIVE_HOME/texmf-local" >> $PROF
+    echo "TEXMFSYSCONFIG $TEXLIVE_HOME/texmf-config" >> $PROF
+    echo "TEXMFSYSVAR $TEXLIVE_HOME/texmf-var" >> $PROF
+    echo "TEXMFVAR $TEXLIVE_HOME/var/texmf-var" >> $PROF
 
-    cd $TEXLIVE_HOME 
+    cd $TEXLIVE_HOME
 
-    ./install-tl --profile=$PROF
+    ./install-tl --repository=$TEXLIVE_INSTALLER_PATH --profile=$PROF
```

Comment:
If the options is called "repository" maybe you should reuse this wording in the readme, and event the dotfile's name.